web/src/topic_popover.js

```diff
@@ -98,42 +98,6 @@ export function initialize() {
                     );
                 });
 
-                $popper.one("click", ".sidebar-popover-unmute-topic", () => {
-                    user_topics.set_user_topic_visibility_policy(
-                        stream_id,
-                        topic_name,
-                        user_topics.all_visibility_policies.UNMUTED,
-                    );
-                    popover_menus.hide_current_popover_if_visible(instance);
-                });
```

Comment:
Just to be clear, this code is now handled by `$popper.on("change", "input[name='sidebar-topic-visibility-select']"`, right?

Reply:
Yes, that's correct!